integration-tests/docker-compose.test.status-go.yml

```
@@ -1,37 +1,61 @@
 services:
   status-go:
+    user: ${INTEGRATION_TESTS_DOCKER_UID}
```

Nice, but maybe this would be more robust:
```suggestion
    user: ${INTEGRATION_TESTS_DOCKER_UID:-$UID}
```

---

Actually I get a bunch of this warnings locally:
```
WARN[0000] The "UID" variable is not set. Defaulting to a blank string. 
```
...which is a bit annoying when there're a lot of them.

---

That's fair. Not all shells set this.